.github/workflows/release.yml

```diff
@@ -257,10 +299,15 @@ jobs:
             org.opencontainers.image.revision=${{ github.sha }}
             org.opencontainers.image.licenses=${{ github.event.repository.license.spdx_id }}
 
+  ####### Prepare the release draft #######
+
   publish-draft-release:
     runs-on: ubuntu-latest
     needs: ["build", "generate-parachain-specs"]
-    if: needs.get-tag-version.outputs.tag-exists == 1
+    if: |
+      github.event_name == 'push' &&
+      github.event.action == 'labeled' &&
+      startsWith(github.ref, 'refs/tags/v')
```

Comment:
Just double checking, is supposed to happen when we tag a commit? Is `labeled` the correct thing for that? It may be, I don't know otherwise. Just surprised to see that since github uses the term "label" to refer to those little labels you can put on PRs or issues.

Reply:
Good catch. I fixed it